ecl/regress/dedupwithbest1.ecl

```diff
@@ -0,0 +1,32 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2016 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and 
+    limitations under the License.
+############################################################################## */
+
+MyRec := RECORD
+	STRING1 Value1;
+	STRING1 Value2;
+END;
+
+SomeFile := DATASET([{'C','G'}, {'C','C'}, {'B','G'}, {'C','B'}, {'A','X'},
+                     {'C','N'}, {'C','T'}, {'C','D'}, {'A','B'} ], MyRec);
+
+Val1Sort := SORT(SomeFile,Value1);
+
+Dedup1   := DEDUP(Val1Sort, Value1, BEST({Value2}) );
+Dedup2   := DEDUP(Val1Sort, Value1, BEST({-Value2}) );
+
+Output(Dedup1);
+Output(Dedup2);
```

Comment:
The tests in testing/rergess/ecl are sufficient - they do not need to be duplicated in ecl/regress.